cluster/evaluator/launch_eval.py

```diff
@@ -178,14 +181,22 @@ def zoo_loop(sgf_dir=None, max_jobs=40):
                           print("{:>30}: {:0.3f} ({:0.3f})".format(modelnum, rate[0], rate[1]))
                         desired_pairs = restore_pairs() or []
                     else:
-                        print("Out of pairs!  Sleeping")
-                        time.sleep(300)
+                        print("Out of pairs.  Sleeping ({} remain)".format(len(r.items)))
+                        time.sleep(600)
                         continue
 
                 next_pair = desired_pairs.pop()  # take our pair off
                 print("Enqueuing:", next_pair)
                 try:
                     same_run_eval(*next_pair)
+                except ApiException as err:
+                    if err.status == 409: # Conflict.  Flip the order and throw it on the pile.
+                        print("Conflict enqueing {}.  Continuing...".format(next_pair))
```

Comment:
I think you should just drop the pair.
1. It's simpler and the pair already has games queued
2. but I'm open to the argument it might change the bell curve

Reply:
i think if we've added a pair that was already in, it was done either by add-top-pairs or add-uncertain-pairs, so there's probably value in running it anyway.